compose/apisix_conf/config.yaml

```diff
@@ -0,0 +1,137 @@
+apisix:
```

Comment:
why we need this file?

Reply:
Is ti only for test? If yes, you can download from apisix repo when run test cases.

Reply:
yes,it is for test
I need to  modified some configurations, such as etcd address and allow_admin

Reply:
I suggest copying this file from the apisix repo, it may expire.
We can use the `sed` command to complete part of the content replacement.